### ISSUE TITLE
自动测试重构剩下的任务列表。

### ISSUE THREAD

**Issue by BBuf** [Oneflow-Inc/oneflow#5905]
- 最新任务列表，基于https://github.com/Oneflow-Inc/oneflow/issues/5549 。
- 如果某些Op有PR或者已经做了，请及时更新表格。
- 项目负责人：张晓雨，梁德澎，姚迟，赵露阳，郑泽康，任天和，李春游，刘沛宏，张申。

| 重构Module                        | 认领人 | reviewer | PR                                                | 备注                                |
| --------------------------------- | :----: | -------- | ------------------------------------------------- | ----------------------------------- |
| oneflow.nn.Softplus               | 张晓雨 |          |                                                   | PyTorch有bug                        |
| oneflow.nn.Mish                   | 郑泽康 |          |                                                   | PyTorch没有                         |
| oneflow.mish                      | 郑泽康 |          |                                                   | PyTorch没有                         |
| oneflow.Tensor.mish               | 郑泽康 |          |                                                   | PyTorch没有                         |
| oneflow.nn.BatchNorm1d            | 赵露阳 |          |                                                   |                                     |
| oneflow.cast                      | 梁德澎 |          |                                                   |                                     |
| oneflow.Tensor.cast               | 梁德澎 |          |                                                   |                                     |
| oneflow.cat                       | 梁德澎 |          |                                                   |                                     |
| oneflow.ones                      | 梁德澎 |          |                                                   |                                     |
| oneflow.zeros                     | 梁德澎 |          |                                                   |                                     |
| oneflow.zeros_like                | 梁德澎 |          |                                                   |                                     |
| oneflow.ones_like                 | 梁德澎 |          |                                                   |                                     |
| oneflow.nn.Module                 |   -    |          |                                                   |                                     |
| oneflow.nn.Parameter              |   -    |          |                                                   |                                     |
| oneflow.nn.Sequential             |   -    |          |                                                   |                                     |
| oneflow.nn.ParameterList          |   -    |          |                                                   |                                     |
| oneflow.nn.ParameterDict          |   -    |          |                                                   |                                     |
| oneflow.nn.ModuleList             |   -    |          |                                                   |                                     |
| oneflow.nn.ModuleDict             |   -    |          |                                                   |                                     |
| oneflow.nn.Conv2d                 | 梁德澎 |          |                                                   |                                     |
| oneflow.nn.ConstantPad2d          | 梁德澎 |          |                                                   |                                     |
| oneflow.nn.ConvTranspose2d        | 梁德澎 |          |                                                   |                                     |
| oneflow.nn.Dropout                | 梁德澎 |          |                                                   |                                     |
| oneflow.equal                     | 任天和 | 张晓雨   | #5599                                             | 有BUG，正在解决                     |
| oneflow.nn.CrossEntropyLoss       | 赵天宇 | 张晓雨   | #5561                                             | 有BUG，未解决                       |
| oneflow.nn.CTCLoss                | 黄振华 |          |                                                   |                                     |
| oneflow.nn.L1Loss                 |        |          |                                                   |                                     |
| oneflow.nn.BCELoss                |        |          |                                                   |                                     |
| oneflow.nn.NLLLoss                |        |          |                                                   |                                     |
| oneflow.nn.KLDivLoss              |        |          |                                                   |                                     |
| oneflow.nn.MSELoss                |        |          |                                                   |                                     |
| oneflow.nn.MarginRankingLoss      |        |          |                                                   |                                     |
| oneflow.nn.BCEWithLogitsLoss      |        |          |                                                   |                                     |
| oneflow.masked_fill               |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.masked_fill        |  姚迟  |          |                                                   |                                     |
| oneflow.sum                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.sum                |  姚迟  |          |                                                   |                                     |
| oneflow.min                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.min                |  姚迟  |          |                                                   |                                     |
| oneflow.max                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.max                |  姚迟  |          |                                                   |                                     |
| oneflow.mul                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.mul                |  姚迟  |          |                                                   |                                     |
| oneflow.mean                      |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.mean               |  姚迟  |          |                                                   |                                     |
| oneflow.sub                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.sub                |  姚迟  |          |                                                   |                                     |
| oneflow.div                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.div                |  姚迟  |          |                                                   |                                     |
| oneflow.var                       |  姚迟  |          |                                                   |                                     |
| oneflow.Tensor.var                |  姚迟  |          |                                                   |                                     |
| oneflow.reciprocal                | 李春游 |          |                                                   |                                     |
| oneflow.Tensor.reciprocal         | 李春游 |          |                                                   |                                     |
| oneflow.add                       | 李春游 |          |                                                   |                                     |
| oneflow.Tensor.add                | 李春游 |          |                                                   |                                     |
| oneflow.sign                      | 李春游 |          |                                                   |                                     |
| oneflow.Tensor.sign               | 李春游 |          |                                                   |                                     |
| oneflow.log                       | 李春游 |          |                                                   |                                     |
| oneflow.Tensor.log                | 李春游 |          |                                                   |                                     |
| oneflow.where                     | 张晓雨 |          |                                                   |                                     |
| oneflow.Tensor.where              | 张晓雨 |          |                                                   |                                     |
| oneflow.gather                    |        |          |                                                   |                                     |
| oneflow.Tensor.gather             |        |          |                                                   |                                     |
| oneflow.nn.Embedding              |        |          |                                                   |                                     |
| oneflow.nn.Upsample               |        |          |                                                   | Pytorh存在一个BUG，不好随机自动测试 |
| oneflow.nn.UpsamplingNearest2d    |        |          |                                                   | Pytorh存在一个BUG，不好随机自动测试 |
| oneflow.nn.UpsamplingBilinear2d   |        |          |                                                   | Pytorh存在一个BUG，不好随机自动测试 |
| oneflow.linalg.norm               |        |          |                                                   |                                     |
| oneflow.Tensor.norm               |        |          |                                                   |                                     |
| oneflow.log1p                     | |          |                                                   |                                     |
| oneflow.Tensor.log1p              | |          |                                                   |                                     |
| oneflow.nn.ReflectionPad2d        |        |          |                                                   |                                     |
| oneflow.topk                      |        |          |                                                   |                                     |
| oneflow.Tensor.topk               |        |          |                                                   |                                     |
| oneflow.nn.GroupNorm              |        |          |                                                   |                                     |
| oneflow.nn.ZeroPad2d              |        |          |                                                   |                                     |
| oneflow.tensor_buffer_to_tensor   |        |          |                                                   |                                     |
| oneflow.tensor_to_tensor_buffer   |        |          |                                                   |                                     |
| oneflow.new_ones                  |        |          |                                                   |                                     |
| oneflow.Conv1d                    |        |          |                                                   |                                     |
| oneflow.ConstantPad1d             |        |          |                                                   |                                     |
| oneflow.chunk                     |        |          |                                                   |                                     |
| oneflow.Tensor.chunk              |        |          |                                                   |                                     |
| oneflow.masked_select             |        |          |                                                   |                                     |
| oneflow.Tensor.masked_select      |        |          |                                                   |                                     |
| oneflow.nn.AdaptiveAvgPool1d      |        |          |                                                   |                                     |
| oneflow.nn.AdaptiveAvgPool3d      |        |          |                                                   |                                     |
| oneflow.adaptive_avg_pool1d       |        |          |                                                   |                                     |
| oneflow.adaptive_avg_pool2d       |        |          |                                                   |                                     |
| oneflow.adaptive_avg_pool3d       |        |          |                                                   |                                     |
| oneflow.fmod                      |        |          |                                                   |                                     |
| oneflow.Tensor.fmod               |        |          |                                                   |                                     |
| oneflow.view                      |        |          |                                                   |                                     |
| oneflow.Tensor.view               |        |          |                                                   |                                     |
| oneflow.flip                      |        |          |                                                   |                                     |
| oneflow.Tensor.flip               |        |          |                                                   |                                     |
| oneflow.nn.functional.interpolate |        |          |                                                   |                                     |
| oneflow.linalg.vector_norm        |        |          |                                                   |                                     |
| oneflow.linalg.matrix_norm        |        |          |                                                   |                                     |
| oneflow.diag                      |        |          |                                                   |                                     |
| oneflow.Tensor.diag               |        |          |                                                   |                                     |
| oneflow.gather_nd                 |        |          |                                                   |                                     |
| oneflow.scatter_nd                |        |          |                                                   |                                     |
| oneflow.nn.image.flip             |        |          |                                                   |                                     |
| oneflow.Tensor.type_as            |        |          |                                                   |                                     |
| oneflow.Tensor.long               |        |          |                                                   |                                     |
| oneflow.bernoulli                 |        |          |                                                   |                                     |
| oneflow.in_top_k                  |        |          |                                                   |                                     |
| oneflow.Tensor.in_top_k           |        |          |                                                   |                                     |
